app/src/main/java/org/mozilla/FirefoxLite/firefoxdownloader/SampleService.java

```diff
@@ -0,0 +1,249 @@
+package org.mozilla.FirefoxLite.firefoxdownloader;
+
+import android.app.Notification;
+import android.app.Service;
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.content.SharedPreferences;
+import android.graphics.Color;
+import android.os.IBinder;
+import android.preference.PreferenceManager;
+import android.util.JsonReader;
+import android.util.Log;
+import android.widget.Toast;
+import androidx.annotation.Nullable;
+import androidx.core.app.NotificationCompat;
+import androidx.lifecycle.LifecycleOwner;
+import androidx.lifecycle.LifecycleService;
+
+import static com.sample.NotificationHelper.channel1ID;
+
+public class SampleService extends LifecycleService {
+    final static String MY_ACTION = "MY_ACTION";
+
+    final String URL1 = "http://mattmahoney.net/dc/enwik8.pmd";
+    private static String dirPath;
+    int downloadIdOne;
+    NotificationHelper mNotificationHelper;
+    MainActivity mainActivity;
+
+    @Override
+    public void onCreate() {
+        super.onCreate();
+        mainActivity=new MainActivity();
```

Comment:
You can't create an Activity like this. What is this for?

Reply:
Just sending another commit for removing this.Initially I used this as a reference for lifecycle owner to be passed to the library.

Reply:
got it